packages/fxa-content-server/tests/server/amplitude.js

```diff
@@ -30,7 +30,7 @@ const amplitude = proxyquire(path.resolve('server/lib/amplitude'), {
   },
   './logging/log': () => logger,
 });
-const APP_VERSION = /^[0-9]+\.([0-9]+)\./.exec(pkg.version)[1];
+const APP_VERSION = pkg.version;
```

Comment:
This will change the app version from, say, `148`, to `1.148.3`.

I think we might want to continue to strip out the initial `1.`?

Reply:
I was indifferent to the `1.` so updated to string it out 👍🏽